src/robusta/cli/main.py

```diff
@@ -331,7 +305,7 @@ def update_config(
 
         try:
             uuid.UUID(values.globalConfig.signing_key)
-        except:
+        except ValueError:
```

Comment:
do all exceptions now need to be explicitly defined?
Is there any other exception that could be thrown here?

Reply:
This is the error from flake8 on bare except and why it is a bad habit:
https://www.flake8rules.com/rules/E722.html

The only expected error here seems to be transformation to `UUID`, that raises `ValueError` on invalid input